app/components/school_information/your_schools/participants_school_search_component.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="govuk-grid-row">
+  <%= component_form_with model: @school_search_form, url: lead_providers_your_schools_path, method: :post do |f| %>
+    <div class="govuk-grid-column-one-quarter index-group">
+      <div class="govuk-form-group govuk-!-margin-bottom-0">
+        <%= f.govuk_text_field :school_name, class: 'govuk-input govuk-!-margin-bottom-3', label: { text: "Search your schools", class: 'govuk-label govuk-label--s' } %>
+        <%= f.hidden_field :cohort_year, value: @selected_cohort.start_year %>
+        <%= f.hidden_field :selected_cohort_id, value: @selected_cohort.id %>
+        <%= f.govuk_submit "Search", classes: 'govuk-button govuk-!-margin-bottom-0' %>
+      </div>
+    </div>
+  <% end %>
+
+  <% if @schools.empty? %>
```

Comment:
Does the job but the pattern here: https://github.com/DFE-Digital/ecf-engage-and-learn/blob/develop/app/views/core_induction_programme/modules/show.html.erb  is definitely better.

Reply:
Done